common/define-grammar.js

```diff
@@ -71,6 +71,7 @@ module.exports = function defineGrammar(dialect) {
       [$.jsx_namespace_name, $._primary_type],
 
       [$.primary_expression, $._parameter_name],
+      [$.primary_expression, $.override_modifier],
```

Comment:
Is this conflict necessary? We don't see conflicts for the other modifiers.

Reply:
This is what I get without it. 🤔 
<img width="631" alt="Screen Shot 2021-08-15 at 12 41 25 AM" src="https://user-images.githubusercontent.com/2881382/129469625-efa6549e-3ba9-4201-b195-fe0693efedf3.png">

Reply:
Should it be defined as a **precedence** instead of conflict? I don't have a strong intuition for either, as a general hint I'd simply take a look at how `readonly` is defined and copy it.

https://github.com/tree-sitter/tree-sitter-typescript/blob/83816f563c8d9d2f1b9c921206a7944d0c5904ad/common/define-grammar.js#L46

Reply:
Cool, I changed it to a precedence instead of conflict, seems to work fine. Thanks for the comment!
<img width="1025" alt="Screen Shot 2021-09-16 at 2 56 16 PM" src="https://user-images.githubusercontent.com/2881382/133684667-6df5cb23-5702-42fa-8feb-07dedcb58f61.png">